mxcubecore/HardwareObjects/Cats90.py

```diff
@@ -141,7 +144,25 @@ def get_sample_address(basket_number, sample_number):
             return ""
 
 
+@unique
+class CatsStates(Enum):
+    """Shutter states definitions."""
+
+    RUNNING = HardwareObjectState.READY, "RUNNING"
+    OF = HardwareObjectState.OFF, "DISABLE"
```

Comment:
Maybe this was supposed to be `OFF` instead of `OF`?

```suggestion
    OFF = HardwareObjectState.OFF, "DISABLE"
```